dotnet/src/Extensions/Planning.StepwisePlanner/StepwisePlanner.cs

```diff
@@ -111,11 +111,15 @@ public Plan CreatePlan(string goal)
 
                 context.Variables.Set("agentScratchPad", scratchPad);
 
-                var llmResponse = (await this._systemStepFunction.InvokeAsync(context).ConfigureAwait(false));
+                SKContext llmResponse;
 
-                if (llmResponse.ErrorOccurred)
+                try
                 {
```

Comment:
As discussed, we probably don't need to catch/throw here any longer.

Reply:
The try/catch is removed.